<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1024</width>
    <height>800</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QPushButton" name="startBtn">
    <property name="geometry">
     <rect>
      <x>450</x>
      <y>20</y>
      <width>93</width>
      <height>28</height>
     </rect>
    </property>
    <property name="text">
     <string>开始</string>
    </property>
   </widget>
   <widget class="QPushButton" name="overBtn">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>610</x>
      <y>20</y>
      <width>93</width>
      <height>28</height>
     </rect>
    </property>
    <property name="text">
     <string>结束</string>
    </property>
   </widget>
   <widget class="QGroupBox" name="groupBox">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>20</y>
      <width>401</width>
      <height>241</height>
     </rect>
    </property>
    <property name="title">
     <string>参数设置</string>
    </property>
    <widget class="QCheckBox" name="logCheckBox">
     <property name="geometry">
      <rect>
       <x>130</x>
       <y>210</y>
       <width>131</width>
       <height>19</height>
      </rect>
     </property>
     <property name="text">
      <string>保存日志纪录</string>
     </property>
     <property name="checked">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QRadioButton" name="suboidRadio">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>120</y>
       <width>100</width>
       <height>19</height>
      </rect>
     </property>
     <property name="layoutDirection">
      <enum>Qt::LeftToRight</enum>
     </property>
     <property name="text">
      <string>扫描OID</string>
     </property>
     <property name="checked">
      <bool>false</bool>
     </property>
    </widget>
    <widget class="QLabel" name="label_2">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>160</y>
       <width>100</width>
       <height>15</height>
      </rect>
     </property>
     <property name="layoutDirection">
      <enum>Qt::LeftToRight</enum>
     </property>
     <property name="text">
      <string>定时时间</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
    <widget class="QDoubleSpinBox" name="timeSpinBox">
     <property name="geometry">
      <rect>
       <x>130</x>
       <y>160</y>
       <width>111</width>
       <height>22</height>
      </rect>
     </property>
     <property name="suffix">
      <string>s</string>
     </property>
     <property name="decimals">
      <number>3</number>
     </property>
     <property name="minimum">
      <double>0.001000000000000</double>
     </property>
     <property name="singleStep">
      <double>0.500000000000000</double>
     </property>
     <property name="value">
      <double>1.000000000000000</double>
     </property>
    </widget>
    <widget class="QLineEdit" name="suboidEdit">
     <property name="geometry">
      <rect>
       <x>130</x>
       <y>110</y>
       <width>260</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>.1.3.6.1.4.1.30966.8.1.1</string>
     </property>
    </widget>
    <widget class="QRadioButton" name="oidRadio">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>70</y>
       <width>100</width>
       <height>19</height>
      </rect>
     </property>
     <property name="layoutDirection">
      <enum>Qt::LeftToRight</enum>
     </property>
     <property name="text">
      <string>固定OID</string>
     </property>
     <property name="checked">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QLabel" name="label">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>30</y>
       <width>100</width>
       <height>15</height>
      </rect>
     </property>
     <property name="layoutDirection">
      <enum>Qt::LeftToRight</enum>
     </property>
     <property name="text">
      <string>IP地址</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
    <widget class="QLineEdit" name="oidEdit">
     <property name="geometry">
      <rect>
       <x>130</x>
       <y>70</y>
       <width>260</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>.1.3.6.1.4.1.30966.8.1.1.1.0</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="ipEdit">
     <property name="geometry">
      <rect>
       <x>130</x>
       <y>30</y>
       <width>260</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>192.168.1.167</string>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="groupBox_2">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>290</y>
      <width>401</width>
      <height>441</height>
     </rect>
    </property>
    <property name="title">
     <string>实时统计</string>
    </property>
    <widget class="QLabel" name="label_8">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>250</y>
       <width>120</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>最长响应时间</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
    <widget class="QLCDNumber" name="okLcd">
     <property name="geometry">
      <rect>
       <x>160</x>
       <y>80</y>
       <width>111</width>
       <height>31</height>
      </rect>
     </property>
    </widget>
    <widget class="QLCDNumber" name="allLcd">
     <property name="geometry">
      <rect>
       <x>160</x>
       <y>30</y>
       <width>111</width>
       <height>31</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="label_5">
     <property name="geometry">
      <rect>
       <x>19</x>
       <y>140</y>
       <width>120</width>
       <height>15</height>
      </rect>
     </property>
     <property name="text">
      <string>超时次数</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
    <widget class="QLabel" name="label_7">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>200</y>
       <width>120</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>平均响应时间</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
    <widget class="QLCDNumber" name="longLcd">
     <property name="geometry">
      <rect>
       <x>160</x>
       <y>240</y>
       <width>111</width>
       <height>31</height>
      </rect>
     </property>
    </widget>
    <widget class="QLCDNumber" name="resLcd">
     <property name="geometry">
      <rect>
       <x>160</x>
       <y>190</y>
       <width>111</width>
       <height>31</height>
      </rect>
     </property>
    </widget>
    <widget class="QLCDNumber" name="errLcd">
     <property name="geometry">
      <rect>
       <x>160</x>
       <y>130</y>
       <width>111</width>
       <height>31</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="label_4">
     <property name="geometry">
      <rect>
       <x>19</x>
       <y>90</y>
       <width>120</width>
       <height>15</height>
      </rect>
     </property>
     <property name="text">
      <string>成功次数</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
    <widget class="QLabel" name="label_3">
     <property name="geometry">
      <rect>
       <x>19</x>
       <y>40</y>
       <width>120</width>
       <height>15</height>
      </rect>
     </property>
     <property name="text">
      <string>请求次数</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
    <widget class="QLabel" name="label_9">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>320</y>
       <width>131</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>测试开始时间</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
    <widget class="QLabel" name="label_10">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>370</y>
       <width>121</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>测试当前时间</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
    <widget class="QDateTimeEdit" name="starttime">
     <property name="enabled">
      <bool>true</bool>
     </property>
     <property name="geometry">
      <rect>
       <x>150</x>
       <y>320</y>
       <width>194</width>
       <height>22</height>
      </rect>
     </property>
     <property name="displayFormat">
      <string>yyyy-MM-dd hh:mm:ss</string>
     </property>
    </widget>
    <widget class="QDateTimeEdit" name="currenttime">
     <property name="enabled">
      <bool>true</bool>
     </property>
     <property name="geometry">
      <rect>
       <x>150</x>
       <y>370</y>
       <width>194</width>
       <height>22</height>
      </rect>
     </property>
     <property name="displayFormat">
      <string>yyyy-MM-dd hh:mm:ss</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_6">
     <property name="geometry">
      <rect>
       <x>280</x>
       <y>200</y>
       <width>40</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>ms</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_11">
     <property name="geometry">
      <rect>
       <x>280</x>
       <y>250</y>
       <width>40</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>ms</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_12">
     <property name="geometry">
      <rect>
       <x>280</x>
       <y>40</y>
       <width>40</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>次</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_13">
     <property name="geometry">
      <rect>
       <x>280</x>
       <y>90</y>
       <width>40</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>次</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_14">
     <property name="geometry">
      <rect>
       <x>280</x>
       <y>140</y>
       <width>40</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>次</string>
     </property>
    </widget>
   </widget>
   <widget class="QTextEdit" name="textEdit">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>430</x>
      <y>70</y>
      <width>581</width>
      <height>661</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="clearBtn">
    <property name="geometry">
     <rect>
      <x>790</x>
      <y>20</y>
      <width>93</width>
      <height>28</height>
     </rect>
    </property>
    <property name="text">
     <string>清空显示</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1024</width>
     <height>26</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <tabstops>
  <tabstop>ipEdit</tabstop>
  <tabstop>oidRadio</tabstop>
  <tabstop>suboidRadio</tabstop>
  <tabstop>timeSpinBox</tabstop>
  <tabstop>logCheckBox</tabstop>
  <tabstop>suboidEdit</tabstop>
  <tabstop>oidEdit</tabstop>
  <tabstop>startBtn</tabstop>
  <tabstop>overBtn</tabstop>
  <tabstop>clearBtn</tabstop>
  <tabstop>textEdit</tabstop>
  <tabstop>currenttime</tabstop>
  <tabstop>starttime</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
